lxd/api_internal.go

```diff
@@ -895,8 +895,8 @@ func internalImportRootDevicePopulate(instancePoolName string, localDevices map[
 	expandedRootName, expandedRootConfig, _ := shared.GetRootDiskDevice(expandedDevices)
```

Comment:
Can this commit ` lxd/db/profiles: Use plain map for ExpandInstanceDevices ` be in a separate PR?